_ml-commons-plugin/algorithms.md

```diff
@@ -65,8 +65,8 @@ Linear regression maps the linear relationship between inputs and outputs. In ML
 
 Parameter | Type   | Description | Default Value
 :--- |:--- | :--- | :---
-learningRate | Double | The rate of speed at which the gradient moves during descent | 0.01
-momentumFactor | Double | The medium-term from which the regressor rises or falls | 0
+learningRate | Double | The gradient descent step size at each iteration when moving toward a minimum of a loss function or optimal value. | 0.01
+momentumFactor | Double | The extra weight factors that accelerate the rate at which the weight is adjusted. This helps move the minimization routine out of local minima.  | 0
 epsilon | Double | The criteria used to identify a linear model | 1.00E-06 
```

Comment:
@Zhangxunmt @ylwu-amzn: Could we copy the definitions for epsilon, beta1, beta2, and decayRate from the Logistic Regression section so they are as follows:
| epsilon | Double | The value for stabilizing gradient inversion | 0.1 |
| beta1 | Double | The exponential decay rates for the moment estimates | 0.9 |
| beta2 | Double | The exponential decay rates for the moment estimates | 0.99 |
| decayRate | Double | The Root Mean Squared Propagation (RMSProp) | 0.9 |

Or do these not mean the same thing in this context? Thanks!

Reply:
Sure, thank you @Zhangxunmt!

Reply:
@greaa-aws: thank you for following up! Please feel free to suggest other documentation improvements so we can implement them.